Fulfillment/UserInterface/01a_List.md

```diff
@@ -0,0 +1,522 @@
+
+The list displays all dispatch notes. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+- *Connection*  
+    Connection to fulfiller.
+
+- *Document number*  
+    Dispatch note number. This number equals the delivery note number in the *Order management* module.
+
+- *Status*  
+    Dispatch note status. The following statuses are available:  
+    - **New dispatch note**  
+        The dispatch note has been newly created.
+    - **Transferred to partner**  
+        The dispatch note has been transferred to the fulfiller.
+    - **Partner confirmed receival**  
+        The fulfiller has confirmed receival of the dispatch note.
+    - **Dispatch note completed**  
+        The dispatch note has been completed.
+    -  **Error**  
+        The dispatch note has given an error while exporting.
+    - **Void**  
+        The dispatch note is void, for instance, because of an order cancellation.
+
+- *Shipment status*  
+    Status of shipment associated with the dispatch note. The following options are available:
+    - ![Cross](../../Assets/Icons/Cross02.png "[Cross]") (Cross)   
+        No shipment has been performed yet.  
+    - ![Circular arrows](../../Assets/Icons/CircularArrows.png "[Circular arrows]") (Circular arrows)   
+        The shipment is in progress.   
+    - ![Double check](../../Assets/Icons/DoubleCheck.png "[Double check]") (Double check)    
+        The shipment has been delivered.
+    - ![Error](../../Assets/Icons/Warning02.png "[Error]") (Error)    
+        The fulfiller has reported a shipment error.
+    - Empty  
+        No information on the shipment status has been received from fulfiller.
+
+- *Description*  
+    Dispatch note description as defined by the user. Numbers, letters or a combination of characters may be used. The information provided in this field can be entered in the search bar to search for a specific dispatch note.
+
+- *Created*  
+    Date and time of the creation.
+
+- *Modified*  
+    Date and time of the last modification.
+
+- *ID*  
+    Dispatch note identification number. The ID number is automatically assigned by the system.
+
+- *Status ID*  
+  Dispatch note status identifier. The following options are available:
+    - **new**
+    - **transferred**
+    - **confirmed**
+    - **completed**
+    - **error**
+    - **void**
+
+- *Carrier*  
+    Carrier identifier.
+
+- *Shipping type*  
+    Type of shipment. The following options are available:  
+    - **STND**  
+        Standard shipping type.
+    - **EXPR**  
+        Express shipping type.
+
+    > [Info] The shipping type can be set up in the *PACKAGING TYPES* tab in the *Settings* menu entry, see [Packaging types](./03b_PackagingTypes.md).
+
+- *Shipping method*  
+    Shipping method (additional services) identifier.
```

Comment:
auch wenn es geändert wird - jetzt bitte für eine Benennung entscheiden; also ohne was in Klammern...